tests/test_hygiene.py

```
@@ -194,6 +196,114 @@ def test_audit_scans_memories_table(self, temp_db):
+            min_score=0.3,
+            scan_all=True,
+            batch_size=2,
+        )
+
+        assert paged.total_scanned == 1
+        assert [c.memory_id for c in paged.candidates] == ["noise1"]
+        assert full.total_scanned == 3
+        assert {c.memory_id for c in full.candidates} == {"noise0", "noise1", "noise2"}
+
+    def test_audit_noise_rejects_invalid_pagination_args(self, temp_db):
+        db_path, beam = temp_db
```

Ruff (0.15.20)

[warning] Unpacked variable `beam` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)